ansible_collections/arista/avd/examples/l2ls-fabric/ansible.cfg

```diff
@@ -0,0 +1,4 @@
+[defaults]
+inventory=inventory.yml
+jinja2_extensions = jinja2.ext.loopcontrols,jinja2.ext.do,jinja2.ext.i18n
+deprecation_warnings=False
```

Comment:
While I understand that these warnings are annoying, we should not hide them, since people will not see important deprecations.

Reply:
removed